src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/LockFileUtils.cs

```diff
@@ -920,7 +920,7 @@ private static PackageDependency GetDependencyVersionRange(LibraryDependency dep
         /// <summary>
         /// Replace excluded asset groups with _._ if they have > 0 items.
         /// </summary>
-        private static void ExcludeItems(LockFileTargetLibrary lockFileLib, LibraryIncludeFlags dependencyType)
+        public static void ExcludeItems(LockFileTargetLibrary lockFileLib, LibraryIncludeFlags dependencyType)
```

Comment:
Does this has to be public? can it be internal?

Reply:
it's being consumed across different assemblies so it needs to be public.